File: CSV Operation/CSV_Operation.cs
using System;
using System.IO;
using System.Text;
using System.Collections.Generic;
using System.Runtime.InteropServices;

public class Class
{
    //针对在unity中实现读写
    private string filePath = "Assets/xxx.csv";
    private StreamReader SR;
    private StreamWriter SW;

    void Main()
	{
        if(File.Exists(filePath))
        {
            //逐行读取
            SR = new StreamReader(filePath);
            string line = SR.ReadLine(); //读取表头
            while(!SR.EndOfStream)
            {
                line = SR.ReadLine();
                string[] row = line.Split(',');
                foreach(string data in row)
                {
                    Console.WriteLine("逐行读取单个为：" + data);
                }
            }
            SR.Close();

            //逐行写入
            SW = new StreamWriter(filePath);
            SW.WriteLine("title, title2, titleN");
            SW.WriteLine("text1, text2, textN");
            SW.Flush();
            SW.Close();
        }
    }
}

File: CSV Operation/CSV_Operation.cpp
#include <iostream>
#include <fstream>
#include <istream>
#include <sstream>
#include <string>
#include <vector>

using namespace std;

string FilePath = "./xxx.csv";

int main(void)
{
	//逐行读取
	ifstream csv_data(FilePath, ios::in);
	string line;

	if (!csv_data.is_open())
	{
		return 0;
	}
	string row;
	istringstream sin;
	while (getline(csv_data, line))
	{
		sin.clear();
		sin.str(line);
		while (getline(sin, row, ','))
		{
			cout << "逐行输出单个为:" << row << endl;
		}
	}
	csv_data.close();

	//行写入
	ofstream write_csv(FilePath, ios::app);
	if (write_csv.is_open())
	{
		//一次endl为一行
		write_csv << "text" << ","
			<< "text_2" << ","
			<< "text_n" << ","
			<< "text_last" << endl;
	}
	write_csv.close();
}
